iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -156,8 +156,9 @@ export class iXBRLViewer {
         else {
             docTitle = "Inline Viewer";
         }
-        if ($('html').attr("lang") === undefined) {
-            $('html').attr("lang", "en-US");
+        if ($('html').attr("xml:lang") !== undefined ) {
+            $(iframe).contents().find('html').attr("xml:lang", $('html').attr("xml:lang"));
+            $('html').removeAttr("xml:lang");
```

Comment:
The `xml:lang` value defined for the report doesn't necessarily match the language of the viewer UI, so keeping it at the root html level seems inappropriate. The report `xml:lang` value is being moved to the iframe HTML element were the report body is also moved.

Reply:
Ah, so this seems like the real fix: we want to set `lang` on the root of the application for accessibility, but to avoid confusing (since there's seemingly no real impact?), we should remove the leftover `xml:lang` from the reparented content document.

ixbrl-viewer only supports two application languages, so the user should only get `lang="en"` or `lang="es"` when looking at the DOM, so we probably need to explain that the user is looking in the wrong place to see the language of their report.